modules/ROOT/pages/build-an-https-service.adoc

```diff
@@ -0,0 +1,145 @@
+= Configure an HTTPS Service
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: mule runtime, arm, https, cloudhub
+
+To help you ensure your data confidentiality, you can deploy your Mule app using an HTTPS based service. Learn how to configure:
+
+* An HTTPS service to deploy your application to CloudHub
+* HTTPS services using API Manager Proxies
+
+If you need to deploy your app locally, see xref:tls-configuration.adoc[TLS Configuration].
+
+== Prerequisites
+
+Before you begin, build a service (such as a simple "Hello World" service) and deploy it.
```

Comment:
It's a bit confusing here that the original plain HTTP service deployment is needed as a prerrequisite. The idea is to modify the original plain HTTP service to HTTPS. Maybe you can add here the full "Hello World" service and to make it a reference for the rest of the doc.

Reply:
In the `Prerequisites` section I added a link to our `[Build an API from Start to Finish](https://docs.mulesoft.com/general/api-led-overview)` tutorial, since the XML tag for the whole app was to big for the starting section of the doc. Let m know if this sounds good.